akka-http2-support/src/test/scala/akka/http/impl/engine/http2/CommonRenderingSpec.scala

```diff
@@ -36,7 +36,7 @@ class ResponseRenderingSpec extends AnyWordSpec with Matchers {
         RawHeader("raw", "whatever"),
         new MyCustomHeader("whatever", renderInResponses = true)
       )
-      ResponseRendering.renderHeaders(headers, builder, None, NoLogging, isServer = true)
+      CommonRendering.renderHeaders(headers, builder, None, NoLogging, isServer = true)
```

Comment:
Can that call be abstracted out? Seems to be mostly the same everywhere.

Reply:
I don't think we can abstract it away. Each test invokes `renderHeaders` with slightly different arguments.